dicom-archive/DICOM/Element.pm

```diff
@@ -156,6 +237,27 @@ sub readFloat {
     return sprintf("%e", $val);
 }
 
+
+=pod
+
+=head3 readSequence($IN, $len)
+
+Reads Sequence.
+
+Three different cases:
+    - implicit Value Representation (VR), explicit length
+    - explicit VR, undefined length, items of defined length (w/end delimiter)
+    - implicit VR, undefined length, items of undefined length
+
+
+INPUTS:
+  - $IN : input file stream
+  - $len: total number of bytes in the field
```

Comment:
total number of bytes to skip, or 0 if all sequences should be skipped until the delimiter 0x00000000 is found.

Reply:
done